mkdocs.yml

```diff
@@ -0,0 +1,116 @@
+# yaml-language-server: $schema=https://squidfunk.github.io/mkdocs-material/schema.json
```

Comment:
https://github.com/gardener/etcd-druid/pull/909#discussion_r1835969619

Also, are the trailing newlines necessary?

Reply:
removed